recipes/install-ua-harvester.rb

```diff
@@ -65,7 +65,7 @@
 cron_d 'ua_harvester' do
   user 'ua_harvester'
   minute '*/2'
-  command %Q(cd /home/ua_harvester/harvester && /usr/bin/run-one ./ua_harvest.py 2>&1 | logger -t info)
+  command %Q(cd /home/ua_harvester/harvester && /usr/bin/run-one ./ua_harvest.py -b 10000 2>&1 | logger -t info)
```

Comment:
oops. this is an unrelated change. just bumping the batch size from the default 1000/fetch.